recipes/scikit-learn-intelex/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "scikit-learn-intelex" %}
+{% set version = "2021.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/intel/scikit-learn-intelex/archive/{{ version }}.tar.gz
+  sha256: 5cfd513245e686d5503ae6ef2d5a17dbf5d0ca41d14b2322a492e1f816ece251
+
+build:
+  skip: true  # [osx]
+  number: 0
+  script: "{{ PYTHON }} setup_sklearnex.py install --single-version-externally-managed --record=record.txt"
+
+requirements:
+  host:
+    - python
+    - setuptools
+  run:
+    - python
+    - daal4py ==2020.3.1
+    - scikit-learn >=0.21
+
+test:
+  commands:
+    - python -c "import sklearnex"
+    - python -c "from sklearnex import patch_sklearn; patch_sklearn()"
+
+about:
+  home: https://intel.github.io/scikit-learn-intelex
+  license: Apache-2.0
```

Because of the third party libraries.
```suggestion
  license: Apache-2.0 AND BSD-2-Clause AND BSD-3-Clause AND MIT
```